hudi-hadoop-mr/src/test/java/org/apache/hudi/hadoop/realtime/TestHoodieRealtimeRecordReader.java

```diff
@@ -491,7 +491,6 @@ public void testSchemaEvolutionAndRollbackBlockInLastLogFile() throws Exception
     writer = writeRollbackBlockToLogFile(partitionDir, schema, "fileid0", commitTime, newCommitTime, "101", 1);
     logFilePaths.add(writer.getLogFile().getPath().toString());
     writer.close();
-    assertTrue("block - size should be > 0", size > 0);
```

Comment:
any reason to remove the assert?

Reply:
Intellij IDEA shows that the size is always larger than 0, so the assertion is always true.

Reply:
okay, got it.